src/components/__tests__/calculate.test.js

```diff
@@ -0,0 +1,19 @@
+import calculate from '../../logics/calculate';
```

Comment:
- [ ] All functionalities and scenarios of the calculate module should be covered using jest. It's from the requirements hence you'll kindly consider the scenarios where **buttonName** `is Zero(0), NotZero(!0), dot(.), equal(=), plus/minus(+/-), `